resources/models/resource.py

```diff
@@ -241,6 +241,21 @@ def free_of_charge(self, is_free):
             return queryset.filter(free_to_use=False, has_pricing=True)
 
 
+class ResourceAccess(models.Model):
+    ACCESS_METHODS = (
```

Comment:
Access method shown on Respa admin/Varaamo.

Because this is M:M (there can be multiple methods for a Resource,e.g. Pincode+Mobile App) I used a model here rather than a simple CharField+choices.